extensions/Worker.Extensions.SignalRService/release_notes.md

```diff
@@ -1,8 +1,6 @@
 ## What's Changed
 
-<!-- Please add your release notes in the following format:
-- My change description (#PR/#issue)
--->
+- Fix SignalR trigger return value not working issue.
```

Comment:
Could you move this below the header (line 5) and update the header to say 1.15.0? Thanks!

Reply:
Labeling for author feedback based on this request.